source/common/http/conn_manager_impl.cc

```diff
@@ -995,10 +1013,23 @@ void ConnectionManagerImpl::ActiveStream::decodeTrailers(ActiveStreamDecoderFilt
   if (!filter) {
     entry = decoder_filters_.begin();
   } else {
-    entry = std::next(filter->entry());
+    if ((*(filter->entry()))->stoppedAll()) {
+      // If filter's iteration_state_ has been stopped, its decodeTrailers() has not be called. Call
+      // it now.
```

Comment:
Similar comment, this only happens during continuation, right? Can you clarify in the comment?

Reply:
Right! Fixed.